pi-manage

```diff
@@ -603,6 +615,20 @@ def dropdb(dropit=None):
                                        checkfirst=True)
     else:
         print("Not dropping anything!")
+    warnings.warn("The function dropdb() will be removed in Version 4.0", DeprecationWarning)
+
+
+@manager.command
+def drop_tables(dropit=None):
+    """
+    This drops all the privacyIDEA database tables.
+    Use with caution! All data will be lost!
+
+    For safety reason you need to pass
+        --dropit==yes
+    Otherwise the command will not drop anything.
+    """
+    dropdb(dropit=None)
```

Comment:
You must only pass "dropit", not "=None".

Reply:
Done in: a5b50932686161a11509b56b4a98ac3665d0d493